src/components/PomodoroButtons/PomodoroButton/PomodoroButton.module.scss

```diff
@@ -0,0 +1,40 @@
+
+.button {
+  width: 6.56rem;
+  height: 3rem;
+  font-family: 'Kumbh Sans', sans-serif;
+  font-size: 0.75rem;
+  font-style: normal;
+  font-weight: 700;
+  line-height: normal;
+  border-radius: 1.66rem;
+  color: #D7E0FF;
+  background: #161932;
+  opacity: 0.4;
+  &:focus {
+    outline: none;
+  }
+}
+
+@media (min-width: 30rem) {
+  .button {
+    width: 7.5rem;
+    font-size: 0.875rem;
+  }
+}
+
+@media (min-width: 48rem) and (orientation:landscape) {
+  .button:hover {
+    opacity: 0.2031;
```

Comment:
Is there a reason for using so specific values instead of more rounded ones like 0.2 (concerns also width and height)?

Reply:
I copied it from figma - these are styles prepared by FrontendMentor team